std/traits.d

```diff
@@ -1480,6 +1484,180 @@ private FunctionAttribute extractAttribFlags(Attribs...)()
     return res;
 }
 
+/**
+Checks whether a function has the given attributes attached.
+
+Params:
+    func = function to check
+    attributes = variadic number of function attributes
+
+Returns:
+    `true`, if the function has the list of attributes attached and `false` otherwise.
+
+See_Also:
+    $(LREF functionAttributes)
+*/
+template hasFunctionAttributes(func...)
```

Comment:
Maybe add an overload for `(Func, strAttributes...)` then? That's best, otherwise at least rename `func` to `args`. (It's pretty annoying alias doesn't accept all types).